src/LibVLCSharp/StreamMediaInput.cs

```diff
@@ -65,10 +67,16 @@ public override int Read(IntPtr buf, uint len)
         {
             try
             {
+#if NETSTANDARD2_1 || ANDROID || APPLE
```

Comment:
would be nice to have only 1 code path here, but I guess this won't be possible

Reply:
I thought about having a HAVE_SPAN that could be declared in the Directory.Build.props